templates/zerver/help/index.md

```diff
@@ -91,7 +89,7 @@ as an **organization**.
 * [Rename a stream](/help/rename-a-stream)
 * Preview a stream (not implemented)
 * [Unsubscribe from a stream](/help/unsubscribe-from-a-stream)
-* [Change who can join a stream](/help/change-who-can-join-a-stream)
+* [Change the accessibility of a stream](/help/change-the-accessibility-of-a-stream)
```

Comment:
Sorry for the late response - had a family-focused weekend. 

Since Tim merged it as "accessibility", I think I was too cautious here :) Thanks for discussing it and double-checking!